packages/calling-component-bindings/src/handlers/createHandlers.ts

```diff
@@ -270,8 +270,8 @@ export const createDefaultCallingHandlers = memoizeOne(
       }
     };
 
-    const onParticipantRemove = (userId: string): void => {
-      call?.removeParticipant(fromFlatCommunicationIdentifier(userId));
+    const onParticipantRemove = async (userId: string): Promise<void> => {
```

Comment:
Not sure how we handle errors in such `async` calls. If this call fails, would the UI show an error?

Reply:
@prprabhu-ms is currently working on the error handling but two things should happen:
1. An onError event should be fired back to the adapter such that Contoso can handle/log/do whatever with that
1. If appropriate show a user facing error message in the banner. These banners make sense for the likes of accessing the camera but will be a judgment call on our part if we want to tell the user certain system errors like these

Reply:
This will be handled by the in-progress error handling.
* Error will be piped to `CallClientState.latestErrors`
* For errors we want to show the user, there will be bindings such that they show up via `ErrorBar`
* Errors will also be saved in adapter state and thrown by adapter method (by doing nothing -- it'll thrown by the adapter method after this PR, the problem is that handling the errors in adapter methods is hard. Likely the method is called from some click handler or useEffect block).

This setup is already in place for Chat. I'm setting it up for Calling in the next couple days.

Reply:
This is almost correct: https://github.com/Azure/communication-ui-library/blob/main/docs/architecture/ErrorReporting.md

Will do another pass once implementation is complete to fix up any places where plans didn't survive contact with the reality.